package.json

```diff
@@ -99,6 +101,12 @@
     "vite-tsconfig-paths": "^4.0.5",
     "vitest": "^0.28.4"
   },
+  "pnpm": {
+    "overrides": {
+      "parse5": "7.1.2",
+      "hast-util-raw": "8.0.0"
+    }
+  },
```

Comment:
Overriding dependencies because of version mismatch, this is a known problem and this is more of a temp fix. We can revisit once maintainers update their dependencies.